src/js/popup/api/gitlab.js

```diff
@@ -21,6 +21,18 @@ function makeRequest(method, url, token) {
   });
 }
 
+function getNextRequest(method, token, linkHeader) {
+  if (!linkHeader) return null;
+
+  const nextLink = linkHeader.split(', ')
+    .map((linkText) => linkText.split('; '))
+    .filter((linkPair) => linkPair[1] == 'rel=\"next\"')
```

Comment:
If only one/none elements are expected - you can use `find` method. Also, add ';' at the end of expression.